<ui version="4.0" >
 <class>teeCheckerClass</class>
 <widget class="QMainWindow" name="teeCheckerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>teeWars Servers Checker</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QPushButton" name="buttonConnect" >
    <property name="geometry" >
     <rect>
      <x>520</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose" >
    <property name="geometry" >
     <rect>
      <x>520</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
    <column>
     <property name="text" >
      <string>1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionSettings" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionClose" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="text" >
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
